src/khoj/configure.py

```diff
@@ -386,3 +386,13 @@ def upload_telemetry():
 def delete_old_user_requests():
     num_deleted = delete_user_requests()
     logger.debug(f"🗑️ Deleted {num_deleted[0]} day-old user requests")
+
+
+@schedule.repeat(schedule.every(17).minutes)
+def wakeup_scheduler():
```

Comment:
Disclaimer.

I'm not actually sure if this is required, but I *think* the `leader` worker won't automatically be aware of new jobs being added to the store unless `wakeup` is explicitly called. There are some background operations that will trigger `wakeup`, but I don't think it's dependable, so this may be a good workaround.